src/UbidotsEthernet.cpp

```diff
@@ -227,7 +227,8 @@ void Ubidots::setDeviceLabel(const char * new_device_label) {
 bool Ubidots::sendAll() {
   /* Assigns the constans as global on the function */
   uint8_t max_retries = 0;
-  uint8_t timeout = 0;
+  //uint8_t timeout = 0; I changed this for a INT timeout
```

Comment:
Thanks for the PR, please delete this comment, it is redundant as the change is reflected in the commit history